app/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.expediagroup.streamplatform</groupId>
     <artifactId>stream-registry</artifactId>
-    <version>0.9.5-SNAPSHOT</version>
+    <version>0.9.6-SNAPSHOT</version>
```

Comment:
No need to version bump. Leave as `0.9.5-SNAPSHOT`

Reply:
done